zerver/lib/import_realm.py

```diff
@@ -1167,6 +1168,9 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
         update_model_ids(GroupGroupMembership, data, "groupgroupmembership")
         bulk_import_model(data, GroupGroupMembership)
 
+    if not UserGroup.objects.filter(realm=realm, is_system_group=True).exists():
+        create_and_add_users_to_system_user_groups(realm, user_profiles)
```

Comment:
Let's add a comment here noting that we expect Zulip server exports to contain these, but that we have this logic here to avoid data exports from other services needing to handle system user groups.